src-tauri/tauri.conf.json

```diff
@@ -29,7 +29,7 @@
         "icons/icon.icns",
         "icons/icon.ico"
       ],
-      "identifier": "com.github.geckoeidechse.flightcore",
+      "identifier": "com.github.r2northstartools.flightcore",
```

Comment:
@emma-miler @RoyalBlue1 just double checking that it's ok for me to use that namespace ^^

Reply:
Emma said: "fine by me"

Blue said: "idk"

so overall good enough for me :P